paddle/operators/logical_op.cc

```diff
@@ -139,15 +139,16 @@ class LogicalOp : public framework::OperatorWithKernel {
       ::paddle::operators::UnaryLogicalOpInferShape<_##op_type##Comment>, \
       ::paddle::framework::EmptyGradOpMaker);
 
-REGISTER_BINARY_LOGICAL_OP(logical_and, "Out = X && Y");
+REGISTER_BINARY_LOGICAL_OP(logical_and, "$$Out = X \\&\\& Y$$");
```

Comment:
What is the reason of putting `\\` before `&`?   I think `\\` will serve as a line breaker.  `\&` will escape and display `&` correctly.

Reply:
This is because it is a C++ string. Since `\` is an escape character in C++ strings, we need to write `\\` if we want a `\` to be a part of the string parsed by latex.

Reply:
I see. Thanks for the clarification.